### DISCUSSION TITLE
skywalking-client-js execution error “Uncaught TypeError: t is not a function”

### DISCUSSION THREAD

**Discussion by aquarius-xu** [apache/skywalking#11403] [Q&A]
Using skywalking-client-js in Vue ran into a problem。
skywalking-client-js: "0.10.0"
Error message：
![image](https://github.com/apache/skywalking/assets/39816202/3cc28429-d370-4195-9f5e-e7c58f38c535)
![image](https://github.com/apache/skywalking/assets/39816202/c2aa7868-dd0c-4966-85f4-c3911226d342)

**Comment by Fine0830** [comment]
How to reproduce this?

**Reply by aquarius-xu** [reply]
This problem is not found in local development, but in production.
I just followed the documentation for the registration command in main.js.
vue version 2.2.6
webpack version 3.6.0
The code: 
![image](https://github.com/apache/skywalking/assets/39816202/527c6718-cba2-4ab8-80ee-928676f2cd7d)

**Comment by Fine0830** [comment]
If you want me to help debug this issue, it is as well to provide a demo to me.

**Reply by aquarius-xu** [reply]
uuid function is reassigned after packaging
![4629535927_120095381001_3FD864A8-8AFD-4e4a-A13C-7DB4D83D8429](https://github.com/apache/skywalking/assets/39816202/a8a76d88-910a-49e2-b1fc-71d66a047d19)

**Reply by Fine0830** [reply]
You can alias a default import for `uuid` in codes.

**Comment by shaolo923** [comment]
Did you solve this problem? I also found it only in production.

**Reply by shaolo923** [reply]
I solved this problem. After downloading the source code and  run build. Js file will be generated. Just introduce it in the index.html of the project you want to use.
